### PR TITLE
WIP | update to latest cumulus

### PR DESCRIPTION
**Pull request by stanly-johnson**
Update xtokens and xcm-support to latest cumulus
Blocked by : https://github.com/paritytech/polkadot/issues/2841

### CODE REVIEW

**Comment by xlc** [comment]
Thanks for the PR. Can you fix the test? You will need to add runtime-benchmarks feature to some deps on Cargo.toml.

File: xtokens/src/mock.rs [hunk 1]
```diff
@@ -123,19 +123,29 @@ decl_test_parachain! {
...
+				pub UnitWeightCost: Weight = 1_000;
+				pub const WeightPrice: (MultiLocation, u128) = (MultiLocation::X1(Junction::Parent), 1_000);
+				pub AllowUnpaidFrom: Vec<MultiLocation> = vec![ MultiLocation::X1(Junction::Parent) ];
+			}
+
 			pub struct XcmConfig;
 			impl XcmConfigT for XcmConfig {
 				type Call = Call;
-				type XcmSender = XcmHandler;
+				type XcmSender = ();
```
Comment (xlc) [inline xtokens/src/mock.rs:139]: why change this?

File: xtokens/src/mock.rs [hunk 2]
```diff
@@ -123,19 +123,29 @@ decl_test_parachain! {
...
 				type AssetTransactor = LocalAssetTransactor;
 				type OriginConverter = LocalOriginConverter;
-				type IsReserve = MultiNativeAsset;
+				type IsReserve = ();
```
Comment (shaunxw) [inline xtokens/src/mock.rs:142]: The `XcmHandler` and `MultiNativeAsset` in config is essential for unit tests, please get them back.

**Review by shaunxw — COMMENTED** [review]
Instead of `opaque::Xcm`, the generic `Xcm` type with `Call` should be used in xcm-support and xtokens.

**Pull request closed without merging**